ledger-core/v2/server/internal/virtual/components.go

```diff
@@ -211,12 +217,12 @@ func initComponents(
 	checkError(ctx, err, "failed to init components")
 
 	// this should be done after Init due to inject
-	pm.AddDispatcher(logicRunner.FlowDispatcher, contractRequester.FlowDispatcher)
+	pm.AddDispatcher(nil, contractRequester.FlowDispatcher)
 
 	return cm, startWatermill(
 		ctx, wmLogger, subscriber, b,
 		nw.SendMessageHandler,
-		logicRunner.FlowDispatcher.Process,
+		nil,
```

Comment:
I'm not sure, but how do you process incoming messages? Why did you drop incoming message handler?

Reply:
It was not a part of an issue, I can't guarantee that server will work or something. 
I'll modify it, but again, I don't think that it'll work.